src/main/java/com/example/kadai7/controller/ComicsController.java

```diff
@@ -0,0 +1,42 @@
+package com.example.kadai7.controller;
+
+import com.example.kadai7.form.ComicForm;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.net.URI;
+import java.util.Map;
+
+@RestController
+public class ComicsController {
+
+    @GetMapping("/comics")
+    public Map<String, String> getComics(@RequestParam String comic) {
+        if (comic.equals("ワンピース")) {
+            return Map.of("name", "ワンピース", "latestVolume", "103巻");
+        } else if (comic.equals("チェンソーマン")) {
+            return Map.of("name", "チェンソーマン", "latestVolume", "11巻");
+        }
+        return null;
+    }
+
+    @PostMapping("/comics")
+    public ResponseEntity<Map<String, String>> postComics(@RequestBody @Validated ComicForm form, UriComponentsBuilder uriBuilder) {
+        URI url = uriBuilder.path("/comics/id").
+                build().
+                toUri();
+        return ResponseEntity.created(url).body(Map.of("message", form.getName() + ":" + form.getLatestVolume() + "巻を登録しました。"));
```

Comment:
```suggestion
        return ResponseEntity.created(url).body(Map.of("message", form.getName() + form.getLatestVolume() + "巻を登録しました。"));
```

```json
{
  "message": "ワンピース1間を登録しました"
}
```
でもいいと思います。
jsonの値の中にコロンを入れるのはkey: valueのコロンと混同してしまうため情報として少し見づらいです。

Reply:
修正いたしました。
![image](https://user-images.githubusercontent.com/102170755/199734844-b1778a6f-1850-4c1d-a181-bf947a7e6a2d.png)